### DISCUSSION TITLE
(ungoogled-chromium) Migrate package

### DISCUSSION THREAD

**Discussion by ghost** [chocolatey-community/chocolatey-packages#2119] [Migrations]
* Why it should be migrated to this repository: There are many times failed when pushing embedded packages from my pc.
It's very stressful since ungoogled-chromium is updated frequently.
* A link to both the package and the repository the source code is located in (if it exists): https://github.com/TakataSanshiro/Chocolatey-Packages/tree/master/automatic/ungoogled-chromium
* Who will be responsible for fixing issues, reviewing changes, and making sure it is kept up to date (ie, fix the updater when it fails): I'm going to update one.

**Comment by AdmiringWorm** [comment]
We have internally agreed to accept migrations of this package, however there will be a few conditions before a PR is accepted.

1. You already mentioned it, but you will need to keep an eye on the package and fix issues that occurs during automatic updates, automated checks on CCR and such.
2. Your PR will need to follow the incoming new Contribution Guidelines, you can see those here: https://github.com/AdmiringWorm/chocolatey-coreteampackages/blob/1754-Update-Guidelines-and-code-to-reflect-new-procedures/CONTRIBUTING.md (Currently opened as a PR, so there may be changes to it at any point).
3. I will create a new issue based on this discussion here, please leave a comment on that issue (so it can be assigned to you) and make sure any PR's created references that issue.
